speaker/layer2_controller_test.go

```diff
@@ -2498,3 +2499,101 @@ func TestClusterPolicy(t *testing.T) {
 		t.Fatalf("All services assigned to speaker1")
 	}
 }
+
+func equalStringSlice(a []string, b []string) bool {
```

Comment:
put this on the bottom, as it's a util function

Reply:
ok